wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -1062,6 +1118,18 @@ extension UserClient {
         }
     }
 
+    public var safeProteusSessionID: ProteusSessionID? {
+        var id: ProteusSessionID?
+        managedObjectContext?.performAndWait {
+            if needsSessionMigration {
+                id = proteusSessionID_V2
+            } else {
+                id = proteusSessionID_V3
+            }
+        }
+        return id
```

Comment:
We don't need to repeat the logic

```suggestion
        return managedObjectContext?.performAndWait { proteusSessionIdentifier }
```